README.md

```diff
@@ -330,8 +345,11 @@ but once available, you can specify which model you want to use when requesting
 This is available for `recommendation`, `sorting` and `campaign` requests:
 
 ```php
-$recommendationCommand = UserRecommendation::create('user-id', 5, 'test-scenario', 1.0, 3600);
-$recommendationCommand->setModelName('alpha');
+$recommendationCommand = UserRecommendation::create('user-id', 'test-scenario');
+    ->setCount(5)
+    ->setRotationRate(1.0)
+    ->setRotationTime(3600)
```

Comment:
Dtto, would be IMHO better to omit those optional setters on leave there just `setModelName`, which is what the example is all about.

Reply:
ok